src/shared/bootspec.c

```diff
@@ -1205,8 +1205,8 @@ BootEntry* boot_config_find_entry(BootConfig *config, const char *id) {
         assert(id);
 
         for (size_t j = 0; j < config->n_entries; j++)
-                if (streq_ptr(config->entries[j].id, id) ||
-                    streq_ptr(config->entries[j].id_old, id))
+                if (strcasecmp(config->entries[j].id, id) == 0 ||
```

Comment:
strcaseeq_ptr() please

Reply:
ahh just found `string-util-fundamental`, thanks for the tip